src/python/base/utils.py

```diff
@@ -308,15 +308,19 @@ def get_file_contents_with_fatal_error_on_failure(path):
   raise errors.BadStateError
 
 
-def get_launch_path_for_script(script_directory, script_filename):
-  """Return launch path for a script."""
-  # Hack for Java Scripts.
-  script_filename = script_filename.replace('.class', '')
-
-  interpreter_path = shell.get_interpreter_for_command(script_filename)
-  script_absolute_path = os.path.join(script_directory, script_filename)
-  script_launch_path = '%s %s' % (interpreter_path, script_absolute_path)
-  return script_launch_path
+def get_execute_command(file_to_execute):
```

Comment:
i think this could be moved to shell.py as well just like get_interpreter_for_command